config/crd/kustomization.yaml

```diff
@@ -8,7 +8,7 @@ resources:
 - bases/metal.ironcore.dev_servers.yaml
 - bases/metal.ironcore.dev_serverbootconfigurations.yaml
 - bases/metal.ironcore.dev_serverclaims.yaml
-- bases/metal.ironcore.dev_servermaintenances.yaml
+- bases/servicemonitor_crd.yaml
```

Comment:
Would need to be removed, if decide to not ship the `ServiceMonitor` CRD.

Reply:
removed